views/score.erb

```diff
@@ -0,0 +1,9 @@
+<form method="post" action="#">
+  <div class="form-group">
+    <label class="shadow" for="word">Enter a Scrabble Word</label>
+    <input type="text" class="form-control" name="word" id="word" placeholder="Enter a word">
+    <input type="checkbox" name="bonus">
+    <label class="shadow" for="bonus">Check for the 7-tile word bonus </label>
```

Comment:
I don't think this label connects to the checkbox input because that input is missing the `id` attribute.